charts/meilisearch/values.yaml

```diff
@@ -35,6 +35,10 @@ auth:
   existingMasterKeySecret: ""
 
 serviceAccount:
+  # You can use a specific pre-created service account instead of
+  # the one created by this chart. Set create to false and enter a name
+  create: true
```

Comment:
Is not the opposite false you use the default account and true to create one?

Reply:
Cheers, I have clarified the variable.

`create: false` prevents this chart from creating a service account.